proposals/0083-relax-entry-constraints.md

```diff
@@ -0,0 +1,62 @@
+---
+simd: '0083'
+title: Relax Entry Constraints
+authors:
+  - Andrew Fitzgerald (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2023-11-02
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+Remove the constraint that transaction entries must not conflict with each
+other.
+
+## Motivation
+
+The current constraint that transaction entries must not conflict with each
+other is unnecessary.
+It is a constraint, that made the current labs-client implementation easier,
+but is not required for the protocol to function correctly.
```

Comment:
Should describe more what is intended behavior incase of transactions writing the same accounts.

Reply:
In an entry all the transactions should be able to run in parallel. If in an entry two transactions are loading same accounts, these transactions cannot be executed in parallel right. Even they will have to maintain ordering of the executing because if one is executed before another in replay then consensus will break. So i guess just explicitly mentioning these cases is important.